docs/variables/youtube.md

```diff
@@ -4,26 +4,36 @@ id: youtube
 
 # $(youtube)
 
-Returns the latest YouTube video for a channel or playlist.
+Returns the latest YouTube video of a channel or the first YouTube video in a playlist.
```

Comment:
i think it should return the latest video in a playlist, and i documented this wrong, but feel free to double check

Reply:
Checked again and it returns the first video in a playlist, not the latest.

Let me know if that's the intended behavior and I'll change it accordingly.

Reply:
That's... odd, thanks for letting me know, it should be latest I think

Reply:
I can't seem to replicate this, do you have an example?

Reply:
should be fixed